examples/ChefCloudNativeBackends.md

```diff
@@ -0,0 +1,311 @@
+# Guide - Setup Chef Infra + Automate on Cloud Native Backends (Azure, AWS and GCP)
```

Comment:
This is all done in AWS but we mention Azure and GCP. Are we intending to tweak with directions for those providers too?

Reply:
apologies, this is another PR I'm working on and this managed to sneak into the PR, I'll remove this and when I make the separate PR for this it will be correct.